networking/hardware_networks/configuring-interface-sysctl-sriov-device.adoc

```diff
@@ -1,12 +1,12 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="configuring-interface-level-sysctl-settings-sriov-device"]
-= Configuring interface-level network sysctl settings for SR-IOV networks
+= Configuring interface-level network sysctl settings and all-multicast mode for SR-IOV networks
 include::_attributes/common-attributes.adoc[]
 :context: configuring-sysctl-interface-sriov-device
 
 toc::[]
 
-As a cluster administrator, you can modify interface-level network sysctls using the tuning Container Network Interface (CNI) meta plugin for a pod connected to a SR-IOV network device.
+As a cluster administrator, you can change interface-level network sysctls and several interface attributes (promiscuous mode, all-multicast mode, MTU, and MAC address) using the tuning Container Network Interface (CNI) meta plugin for a pod connected to a SR-IOV network device.
```

Comment:
```suggestion
As a cluster administrator, you can change interface-level network sysctls and several interface attributes such as promiscuous mode, all-multicast mode, MTU, and MAC address by using the tuning Container Network Interface (CNI) meta plugin for a pod connected to a SR-IOV network device.
```
[See ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses) for avoiding parentheticals.